compiler/CodeGenVisitor.cpp

```diff
@@ -222,14 +237,21 @@ antlrcpp::Any CodeGenVisitor::visitReturnValue(ifccParser::ReturnValueContext *c
 
 antlrcpp::Any CodeGenVisitor::visitInit(ifccParser::InitContext *ctx)
 {
+	cout << "\t#visit init" << endl;
```

Comment:
```suggestion
	cout << "\t# visit init" << endl;
```
Else the vscode parser doesn't like it